machine-types_5b24032b0428632c466af399.md

```diff
@@ -1,11 +1,18 @@
-A  **machine type** specifies a particular collection of virtualized
+
+* [Overview](#overview)
+* [Linux machine types](#linux-machine-types)
+* [Apple machine type](#apple-machine-type)
+
+## Overview
+
+A **machine type** specifies a particular collection of virtualized
 hardware resources available to a virtual machine (VM) instance,
 including the memory size, virtual CPU count, and disk.
 
-This guide describes the available machine types. For using machine
-types in pipelines see pipeline [agent documentation][1].
+This guide describes the available machine types. In order to use the various
+machine types in pipelines see the documentation of the [agent property][1].
```

Comment:
I liked the previous statement a bit more:

```
see pipeline [agent documentation][1]
```

`agent property` is a bit abstract and not so friendly to the reader.

Reply:
Will change that and merge it. Thanks